lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
@snusnu The idea behind `request` is to produce a "real" OO interface of an HTTP request. Not a primitive obsessive rack env. `snusnu/cookie` does do a very good job to present the cookie component of an HTTP request in a OO API, so yes. I'd consider a merge.